dev/dll/SharedHelpers.cpp

```diff
@@ -143,12 +143,11 @@ bool SharedHelpers::IsFrameworkElementInvalidateViewportAvailable()
     return s_isFrameworkElementInvalidateViewportAvailable;
 }
 
-bool SharedHelpers::IsApplicationViewGetDisplayRegionsAvailable()
+bool SharedHelpers::IsApplicationViewGetSpanningRectsAvailable()
 {
-    static bool s_isApplicationViewGetDisplayRegionsAvailable =
-        Is19H1OrHigher() ||
-        winrt::ApiInformation::IsMethodPresent(L"Windows.UI.ViewManagement.ApplicationView", L"GetDisplayRegions");
-    return s_isApplicationViewGetDisplayRegionsAvailable;
+    static bool s_isApplicationViewGetSpanningRectsAvailable =
+        winrt::ApiInformation::IsMethodPresent(L"Windows.UI.ViewManagement.ApplicationView", L"GetSpanningRects");
+    return s_isApplicationViewGetSpanningRectsAvailable;
```

Comment:
ApiInformation checks aren't cheap, can we just use the QueryInterface as the gateway and delete this helper? If we can't QI for the special interface then it doesn't matter what the metadata says.

Reply:
sure!